OCR/frontend/src/types/models.ts

```diff
@@ -0,0 +1,120 @@
+import {Shape} from "react-image-label";
+
+
+// -----------------------------------------------------------------------------
+// Report Vision Models
+// -----------------------------------------------------------------------------
+
+type Base64<imageType extends string> = `data:image/${imageType};base64${string}`
+
+interface Template {
+    id: string;
+    name: string;
+    description: string;
+    labName: string
+    pages: Page[];
+    status: TemplateStatus;
+    createdBy: User;
+    updatedBy: User | undefined;
+    createdAt: Date;
+    lastUpdated: Date | undefined;
+    organization: Organization;
+}
+
+type TemplateStatus = "Completed" | "In Progress" | "Deprecated"
+
+interface Label {
+    key: string;
+    color: string;
+    humanReadableName: string;
+    required: boolean;
+    createdAt: Date;
+    updatedAt: Date | undefined;
+    createdBy: User;
+    updatedBy: User | undefined;
+}
+
+interface Page {
+    id: string;
+    baseImage: Base64<"png"|"jpg"|"jpeg">;
+    segmentationTemplate: Base64<"png"|"jpg"|"jpeg">;
```

Comment:
only `png`.  jpeg doesnt preserve the rgb value that you set since its a lossy compression.

Reply:
I also removed it from `baseImage` for when we're processing an image.